edx_sga/templates/staff_graded_assignment/show.html

```diff
@@ -13,7 +13,7 @@
       <p>No file has been uploaded.</p>
     <% } %>
     <% if (graded) { %>
-      <p>Your score is <%= graded.score %> / <%= max_score %><br/>
+      <p>Your score is <%= (graded.score / max_score) * weight %> / <%= weight %><br/>
```

Comment:
@pdpinch @carsongee  If weight is Zero (0), it will show grade 0 / 0 is this ok?

![zero](https://cloud.githubusercontent.com/assets/10431250/7985025/1695481a-0ae7-11e5-8033-af348aa0c7cc.png)